src/SelfScreener/EmergencySymptomsQuestions.spec.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react"
+import { fireEvent, render } from "@testing-library/react-native"
+import { useNavigation } from "@react-navigation/native"
+
+import { SelfScreenerContext } from "../SelfScreenerContext"
+import EmergencySymptomsQuestions from "./EmergencySymptomsQuestions"
+import { EmergencySymptom } from "./selfScreener"
+import { factories } from "../factories"
+import { SelfScreenerStackScreens } from "../navigation"
+
+jest.mock("@react-navigation/native")
+describe("EmergencySymptomsQuestions", () => {
+  describe("updating symptoms", () => {
+    it("calls the update function", () => {
```

Comment:
How do we feel about not nesting the it and the describe here? seems like we have one example. What do we think about 
```suggestion
  it("allows the user to update the symptoms", () => {
```